README.md

```diff
@@ -5,9 +5,12 @@ The purpose of this test is 3 fold:
 * Assess how comfortable you are with writing automation tests and the approach taken to accomplish it
 
 ## 📕 What you need to do:
-* Sign up at https://gender-api.com/
-* Read the api docs: https://gender-api.com/en/api-docs
-* Determine how you want to test the end points via automation tests
+Write tests for the following cases at https://gorest.co.in/:
```

Comment:
Perhaps rephrase it a bit:
Using the testing site: https://gorest.co.in/, write automation tests to test for the following cases: